README.md

```diff
@@ -26,18 +28,24 @@ AWS_CREDENTIALS: |
 NPM_INSTALL_TOKEN: 123456789
 ```
 
-Note that the value for `AWS_CREDENTIALS` is a multi line string (with spaces and no tabs).
+> Note that the value for `AWS_CREDENTIALS` is a multi line string (with spaces and no tabs).
 
-- Run the following command from the directory that contains the properties file:
+Run the following command from the directory that contains the properties file, where:
 
 ```sh
-$ setAllVars --url <gitlab-project-url> --token <gitlab-token>
+$ setAllVars --token <gitlab-token> --url <gitlab-project-url>
 ```
 
-where `gitlab-project-url` is the url to view your project on gitlab, e.g. https://gitlab.com/gitlab-org/gitlab-ce
```

Comment:
I think this statement still adds some value, because it tells people to use the gitlab url on the browser to view it, not the one you use to clone the project. What do you reckkon?